types/base/api.d.ts

```diff
@@ -39,19 +43,30 @@ type ImportOrderOptions = {
     meta?: string[];
 }
 
-type RequestOptions = {
+export type RequestOptions = {
     url?: string;
     method?: string;
     params?: Record<string, unknown>,
     headers?: Record<string, unknown>,
     auth?: boolean
 }
 
+export type GetRequestOptions = RequestOptions & {
+    params?: {
+        filters?: string[];
+        sort?: string[];
+        skip?: number;
+        limit?: number;
+    }
+}
+
 export declare class RipeAPI {
     key?: string;
 
     constructor(options?: unknown);
     authKeyP(key: string, options?: RequestOptions): Promise<void>;
-    importOrderP(ffOrderId: string, options?: ImportOrderOptions): Promise<void>;
+    importOrderP(ffOrderId: string, options?: ImportOrderOptions): Promise<Order>;
+    getOrdersP(options?: GetRequestOptions): Promise<Order[]>;
+    deleteOrderP(number: number, options?: RequestOptions): Promise<string|null>;
```

Comment:
When does it return `null`? Can't you `string?`?

Reply:
In return types it looks like you can't use nullable:
![image](https://user-images.githubusercontent.com/16060539/106460590-7a4c8900-648b-11eb-9f3d-01a09ff960f4.png)
Furthermore, it says this is either `string` or `null`, never `undefined`.

Looks like the return is wrong regardless as it is not a string. I need to take a deeper look.